sympy/core/power.py

```diff
@@ -124,6 +124,15 @@ class Pow(Expr):
     | (-oo)**oo    | nan     |                                               |
     | (-oo)**-oo   |         |                                               |
     +--------------+---------+-----------------------------------------------+
+    | oo**I        | nan     |                                               |
+    | (-oo)**I     |         |                                               |
+    +------------------------------------------------------------------------+
+    | oo**(1+I)    | zoo     |                                               |
+    | (-oo)**(1+I) |         |                                               |
+    +------------------------------------------------------------------------+
+    | oo**(-1+I)   | 0       |                                               |
+    | (-oo)**(-1+I)|         |                                               |
```

Comment:
Instead  of `oo**z` as general expression and the using `re(z) < 0`, `re(z) > 0` and `re(z) = 0`, since in just above for `oo**(-1)` is written as a general example for all the expressions like `oo**(-5), oo**(-6)` and so on.... . So following the trend, i find current way of doing it better.
@leosartaj What you say? Since initially your raised this point.